apps/i18n/common/en_us.json

```diff
@@ -979,6 +979,7 @@
   "lastUpdatedNoTime": "Last Updated:",
   "lastUpdatedWithTime": "Last updated {time}",
   "learnMore": "Learn more",
+  "learnMoreWithPeriod": "Learn more.",
```

Comment:
Ya I agree it seemed weird but I never know with translations if somehow the punctuation matters so I felt like this was safer since Amanda wanted the period here.

Reply:
@fontie715 ^